content/blog/2021-01-22-holiday-surveys.Rmd

```diff
@@ -78,6 +75,8 @@ library(dplyr)
 library(grid)
 library(gridExtra)
 
+options(covidcast.auth = Sys.getenv("API_KEY")) # for more on API keys, see: https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html
```

Comment:
There are rendering problems with this code block, see [preview](https://deploy-preview-891--cmu-delphi-main.netlify.app/blog/2021/02/02/home-for-the-holidays-the-impact-of-us-holidays-on-social-behaviors-and-preventative-measures/#travel-and-other-social-behaviors-during-us-holidays) and [live](https://delphi.cmu.edu/blog/2021/02/02/home-for-the-holidays-the-impact-of-us-holidays-on-social-behaviors-and-preventative-measures/#travel-and-other-social-behaviors-during-us-holidays).  Later code blocks do not seem to suffer from this.

Reply:
fixed. [fe5e6ff](https://github.com/cmu-delphi/www-main/pull/891/commits/fe5e6ff0ee88eecf8f5752ae1f9f3514f68cf4fe)